expression/builtin_math_test.go

```diff
@@ -191,3 +191,25 @@ func (s *testEvaluatorSuite) TestCRC32(c *C) {
 		c.Assert(v, testutil.DatumEquals, t["Ret"][0])
 	}
 }
+
+func (s *testEvaluatorSuite) TestConv(c *C) {
+	defer testleak.AfterTest(c)()
+	tbl := []struct {
+		Arg []interface{}
+		Ret interface{}
+	}{
+		{[]interface{}{"a", 16, 2}, "1010"},
+		{[]interface{}{"6E", 18, 8}, "172"},
+		{[]interface{}{"-17", 10, -18}, "-H"},
+		{[]interface{}{nil, 10, 10}, nil},
+		{[]interface{}{"+18aZ", 7, 36}, 1},
```

Comment:
`10+'10'+'10'+X'0a'` will handle firstly by opfunction, add in here will no work.